man/file-hierarchy.xml

```diff
@@ -127,20 +127,23 @@
         <term><filename>/tmp/</filename></term>
         <listitem><para>The place for small temporary files. This directory is usually mounted as a
         <literal>tmpfs</literal> instance, and should hence not be used for larger files. (Use
-        <filename>/var/tmp/</filename> for larger files.) Since the directory is accessible to other users of
-        the system, it is essential that this directory is only written to with the <citerefentry
-        project='man-pages'><refentrytitle>mkstemp</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
-        <citerefentry
-        project='man-pages'><refentrytitle>mkdtemp</refentrytitle><manvolnum>3</manvolnum></citerefentry> and
-        related calls. This directory is usually flushed at boot-up. Also, files that are not accessed within
-        a certain time are usually automatically deleted. If applications find the environment variable
-        <varname>$TMPDIR</varname> set, they should prefer using the directory specified in it over directly
-        referencing <filename>/tmp/</filename> (see <citerefentry
+        <filename>/var/tmp/</filename> for larger files.) This directory is usually flushed at boot-up. Also,
+        files that are not accessed within a certain time may be automatically deleted.</para>
+
+        <para>If applications find the environment variable <varname>$TMPDIR</varname> set, they should use
+        the directory specified in it instead of <filename>/tmp/</filename> (see <citerefentry
         project='man-pages'><refentrytitle>environ</refentrytitle><manvolnum>7</manvolnum></citerefentry> and
         <ulink url="http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html#tag_08_03">IEEE
-        Std 1003.1</ulink> for details). For further details about this directory, see <ulink
-        url="https://systemd.io/TEMPORARY_DIRECTORIES">Using /tmp/ And /var/tmp/
-        Safely</ulink>.</para></listitem>
+        Std 1003.1</ulink> for details).</para>
+
+        <para>Since <filename>/tmp</filename> is accessible to other users of the system, it is essential
```

Comment:
we usually try to suffix paths to dirs with / to make it clear we are talking about a dir here, and the paragraph here mostly gets this right, hence do that here too?

Reply:
this wasn't addressed?